test/test_grade/test_integration.py

```diff
@@ -138,6 +139,9 @@ def test_notebooks_with_pdfs(expected_points):
     # read the output and expected output
     df_test = pd.read_csv("test/final_grades.csv")
 
+    #remove points possible row
+    df_test = df_test[df_test['file'] != POINTS_POSSIBLE_LABEL]
```

Comment:
Please update this test to also test that the points possible row has the correct values.

Reply:
Removed these lines to test the points possible row has correct values